dags.yaml

```diff
@@ -168,6 +168,23 @@ bqetl_subplat:
   tags:
     - impact/tier_1
 
+bqetl_subplat_hourly:
+  schedule_interval: 30 * * * *
```

Comment:
Is it possible to make this 45 so that it's aligned with `bqetl_subplat`?  Just a nit but it would make the negative execution delta easier to reason about

Reply:
It's possible, but here's my reasoning for picking the schedule I did:
- With some of the hourly ETLs writing to date-partitioned tables, in between the first and second run of each day the data lag for such tables will be greater than one hour.  With the current schedule that lag should be at most ~1.5 hours (half an hour more than usual), while if we used `45 * * * *` that max lag would be longer (~1.75 hours, or 45 minutes more than usual).
  - @chelseybeck how much would the max data lag matter for Braze?
- By starting the hourly ETLs 15 minutes before the daily ETLs that select from some of those hourly ETL tables, we allow the hourly ETLs time to potentially complete before the downstream daily ETLs start running.  With the current execution deltas the daily ETLs aren't depending on the most recent hourly DAG run, so they won't block on waiting for the possibly-still-running hourly ETLs as they're modifying the tables some daily ETLs are selecting from.  This isn't an absolute dealbreaker, but I think ideally we'd want the hourly ETLs to normally finish before the daily ETLs run to minimize confusion about what state those hourly tables were in when the daily ETLs were selecting from them.
  - Following this logic, if the hourly DAG runs end up taking appreciably longer than 15 minutes we may want to change this schedule to something like `15 * * * *` to give those more time to complete before the daily ETLs run.
  - Or alternatively, I could change the execution deltas for cases where a daily ETL depends on an hourly ETL that _doesn't_ write to a specific date-partition so it depends on the most recent run. However, that makes calculating/managing the time deltas a bit more complicated.  What do you think @BenWu?

Reply:
And another angle to this is the execution deltas that will need to be calculated/managed for the `bqetl_braze` DAG tasks that will depend on `bqetl_subplat_hourly` tasks.  The [`bqetl_braze` DAG's schedule](https://github.com/mozilla/bigquery-etl/blob/80de43a57f198f58a5605dcafce5445c2dd442f0/dags.yaml#L1544) is currently `30 4,12,20 * * *`.

@chelseybeck what's your plan for how the `bqetl_braze` DAG will ultimately be scheduled?  Will it end up running hourly?

Reply:
yes, our plan is to run it hourly as soon as we can get ctms data that frest :)